### ISSUE TITLE
DOTCOM-25388 - Accessibility: Missing focus indicator on first link inside cards

### ISSUE THREAD

**Issue by rofe** [adobe/theblog#329]
Ex URL: https://blog.adobe.com/

**Reproduction Steps**

1. Tab through the above given URL using keyboard.
2. Once you reach a card using keyboard, there is not focus indicator around the first link that wraps the image.
3. Now use CSS to to show focus indicator.
4. This focus indication is missing on all links that wrap images in cards (Applies to all 3 kinds of cards)

**Expected Behavior:**

All links in cards that show images should have a visible focus indicator.

**Actual Behavior:**

Links in cards that show images have no visible focus indicator.

**WCAG Success Criteria**

[Focus visible](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)
